contracts/adapters/Onboarding.sol

```diff
@@ -59,30 +60,31 @@ contract OnboardingContract is
         address tokenAddr;
     }
 
-    mapping(address => OnboardingConfig) public configs;
+    mapping(address => mapping(address => OnboardingConfig)) public configs;
     mapping(address => mapping(uint256 => ProposalDetails)) public proposals;
 
     function configureDao(
         DaoRegistry dao,
+        address tokenAddrToMint,
         uint256 chunkSize,
         uint256 sharesPerChunk,
         address tokenAddr
     ) external onlyAdapter(dao) {
-        configs[address(dao)].chunkSize = chunkSize;
-        configs[address(dao)].sharesPerChunk = sharesPerChunk;
-        configs[address(dao)].tokenAddr = tokenAddr;
+        require(chunkSize > 0, "chunkSize must be higher than 0");
```

Comment:
I thought about that ... I think that it isn't really an issue if sharesPerChunk is 0 and might make sense if you want to create a donation onboarding (where ppl can send money but get 0 shares)

Reply:
I remember we have discussed that the donation/tip would be supported by the DAORegistry by default, mainly because we don`t need a proposal for that. If someone sends money to the DAORegistry contract it will accept, we don`t have the `receive() external payable` revert call.

Reply:
sure, the question here is more whether we NEED to do this test. I think we don't. chunkSize needs to be > 0 but sharesPerChunk might have a use case with 0 that we don't think about right now and I don't see any issue / security risk with it being 0